### PR TITLE
#2512 Properly handle 3WT

### PR DESCRIPTION
**Pull request by barondim**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [x] unit tests and non-regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

**Comment by barondim** [comment]
- [ ] Ne pas merger, des tests ne passent pas.

**Comment by barondim** [comment]
Merge this task with https://github.com/dynawo/dynaflow-launcher/issues/549

File: dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp [hunk 1]
```diff
@@ -679,6 +680,7 @@ DataInterfaceIIDM::convertThreeWindingsTransformers(powsybl::iidm::ThreeWindings
     }
     network_->addTwoWTransformer(fictTwoWTransf);
     components_[fictTwoWTransf->getID()] = fictTwoWTransf;
+    fict2wtIDto3wtID_.insert({fictTwoWTransf->getID(), ThreeWindingTransformer.getId()});
```
Comment (rosiereflo) [inline dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp:683]: I know it's should have been already done, but could you please add a unit test for those 2 methods?

File: dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.h [hunk 2]
```diff
@@ -471,6 +471,8 @@ class DataInterfaceIIDM : public DataInterfaceImpl {
   boost::unordered_map<std::string, std::vector<boost::shared_ptr<CalculatedBusInterfaceIIDM> > > calculatedBusComponents_;  ///< calculatedBus per voltageLevel
   boost::shared_ptr<ServiceManagerInterfaceIIDM> serviceManager_;  ///< Service manager
 
+  std::unordered_map<std::string, std::string> fict2wtIDto3wtID_;                                  ///< map of fictitious 2WTs and they're associated 3WT
```
Comment (rosiereflo) [inline dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.h:474]: their

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**